packages/libs/components/src/plots/BipartiteNetwork.tsx

```diff
@@ -107,18 +109,14 @@ function BipartiteNetwork(
   // nodes based on their column, then will use their order in the column
   // (given by columnXNodeIDs) to finally assign the coordinates.
```

Comment:
I checked out some of the other comments in this file and lots make references to the columns. To prevent confusion, we should update the wording to explain that it's a bipartite network, so it has two partitions of nodes. By default we'll render the nodes in a two-column layout. More options for layouts are a later feature.